src/main/java/profile_screens/ProfileScreen.java

```diff
@@ -1,12 +1,13 @@
 package profile_screens;
```

Comment:
I changed one of my methods to throw an exception, so here I just surrounded the block that called that method with try-catch